.github/workflows/publish-experimental.yml

```diff
@@ -32,9 +32,9 @@ jobs:
 
       # create an experimental version at this SHA (vX.X.X-experimental-4742ef14cc0ea07e3569eee56899937452a55a9c)
       - name: Version
-        run: npm version "$(npm run current-version -s)-experimental-$GITHUB_SHA" --no-git-tag-version
+        run: npm version "$(npm run current-version -s)-experimental-$(git rev-parse HEAD)" --no-git-tag-version
```

Comment:
because this isn't triggered by a PR (but by a comment) GITHUB_SHA actually just defaults to the HEAD ref of main.